ksqldb-common/src/main/java/io/confluent/ksql/util/KsqlConfig.java

```diff
@@ -138,7 +138,17 @@
   public static final String KSQL_KEY_FORMAT_ENABLED = "ksql.key.format.enabled";
```

Comment:
i've merged the other PR, so if you want to we can rename this (or just leave it as is, i'm OK with that too)

Reply:
I'll leave it for now since the old name is already embedded in some historic plans (with value false), no need to introduce yet another config to clutter the history. Plus this flag is going away soon ;D